CMakeLists.txt

```diff
@@ -542,6 +549,38 @@ if(HAVE_FPGA)
   ExternalProject_Get_Property(FPGA_library binary_dir)
   target_link_libraries(opmsimulators PUBLIC ${binary_dir}/fpga_lib_alveo_u280.a)
 endif()
+if (USE_DAMARIS_LIB) # User requested Damaris support
+        if (Damaris_FOUND)
```

Comment:
please make a wrapper find module instead of putting all of this in the main cmakelists.